config/server-vars.yml

```diff
@@ -57,13 +58,18 @@ EDXAPP_FEATURES:
 
 
 # Configure email addresses
-EDXAPP_CONTACT_EMAIL: "%%PLATFORM_EMAIL%%"
-EDXAPP_TECH_SUPPORT_EMAIL: "{{ EDXAPP_CONTACT_EMAIL }}"
-EDXAPP_BUGS_EMAIL: "{{ EDXAPP_CONTACT_EMAIL }}"
-EDXAPP_DEFAULT_FROM_EMAIL: "{{ EDXAPP_CONTACT_EMAIL }}"
-EDXAPP_DEFAULT_FEEDBACK_EMAIL: "{{ EDXAPP_CONTACT_EMAIL }}"
-EDXAPP_DEFAULT_SERVER_EMAIL: "{{ EDXAPP_CONTACT_EMAIL }}"
-EDXAPP_BULK_EMAIL_DEFAULT_FROM_EMAIL: "{{ EDXAPP_CONTACT_EMAIL }}"
+EDXAPP_BUGS_EMAIL: "bugs@openedx.microsoft.com"
```

Comment:
Suggest adding these hard-coded values to env configuration so we can use different accounts for bvt/dev/int/prod.